src/main/java/it/gov/pagopa/pu/workflow/wf/debtposition/custom/activity/CancelReductionExpirationScheduleActivity.java

```diff
@@ -0,0 +1,11 @@
+package it.gov.pagopa.pu.workflow.wf.debtposition.custom.activity;
+
+import io.temporal.activity.ActivityInterface;
+import io.temporal.activity.ActivityMethod;
+
+@ActivityInterface
+public interface CancelReductionExpirationScheduleActivity {
+
+  @ActivityMethod
+  void cancelScheduling(String workflowId);
```

Comment:
maybe it could be better to accept the debtPositionId and let the activity to build the workflowId, what do you think about it?
```suggestion
  void cancelScheduling(Long debtPositionId);
```

Reply:
Could be, i was thinking about doing it directly in the WF class, but is fine here too